src/parser.rs

```diff
@@ -334,7 +334,7 @@ pub struct LocalVarDef {
 /// This is a "dense" segment, which must be fully rebuilt in order to make any change.  We may in
 /// the future have an "unpacked" segment which is used for active editing, as well as a "lazy" or
 /// "mmap" segment type for fast incremental startup.
-#[derive(Debug)]
+#[derive(Debug,Default)]
```

Comment:
Not crazy about Default implementations for exported types that produce invalid values.  All valid Segments must have at least one statement, etc.